src/components/cc-pricing-header/cc-pricing-header.smart.md

```diff
@@ -35,3 +35,44 @@ title: '💡 Smart'
   <cc-pricing-header></cc-pricing-header>
 </cc-smart-container>
 ```
+
+## 📄 Notes
+
+Pricing components rely on nested `cc-smart-container` components.
+
+1. There is a global `<cc-smart-container>` wrapping all pricing components to provide shared context,
+2. There are individual `<cc-smart-container>` wrapping each `<cc-pricing-product>` & `<cc-pricing-product-consumption>` component slotted within the `<cc-pricing-page>` to provide context specific to each product.
+
+The typical HTML for a basic `cc-pricing-page` implementation looks like this:
+```html
+<cc-smart-container context='{
```

Comment:
same question as the `pricing-estimation`

Reply:
Nice catch, forgot about that, thanks!